### ISSUE TITLE
1.11.9 Uncaught TypeError: Cannot read property 'onDidDestroy' of undefined

### ISSUE THREAD

**Issue by emileber** [steelbrain/linter#1182]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Unknown Windows Version
**Thrown From**: [linter](https://github.com/atom-community/linter) package, v1.11.9

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidDestroy' of undefined

```
At /C:/Users/emile.bergeron.CAMINO/.atom/packages/linter/lib/linter-views.js:123

TypeError: Cannot read property 'onDidDestroy' of undefined
    at LinterViews.renderBubble [as _renderBubble] (C:/Users/emile.bergeron.CAMINO/.atom/packages/linter/lib/linter-views.js:85:20)
    at file:///C:/Users/emile.bergeron.CAMINO/.atom/packages/linter/lib/helpers.coffee:23:18
```

### Commands

```
     -3:29.7.0 temporary-root:enter-root-mode (div.header.list-item)
     -0:18.1.0 tree-view:add-file (div.header.list-item)
  3x -0:15.7.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:15.1.0 core:undo (atom-text-editor.editor.mini.is-focused)
     -0:14.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:09.4.0 core:copy (atom-text-editor.editor.is-focused)
     -0:08.2.0 core:paste (atom-text-editor.editor.is-focused)
     -0:06.8.0 docblockr:parse-inline (atom-text-editor.editor.is-focused)
     -0:05.7.0 docblockr:parse-tab (atom-text-editor.editor.is-focused)
     -0:05.6.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:05.6.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:05.6.0 docblockr:parse-tab (atom-text-editor.editor.is-focused)
     -0:05.6.0 editor:indent (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "autoHideMenuBar": true,
    "disabledPackages": [
      "sass-compiler"
    ],
    "packagesWithKeymapsDisabled": [
      "autoclose-html"
    ],
    "projectHome": "C:\\git"
  },
  "linter": {
    "showErrorPanel": false
  }
}
```

### Installed Packages

```coffee
# User
autoclose-html, v0.23.0
color-picker, v2.1.1
docblockr, v0.7.3
emmet, v2.4.3
file-icons, v1.7.14
foldername-tabs, v0.1.11
highlight-selected, v0.11.2
linter, v1.11.9
linter-jshint, v2.1.0
linter-php, v1.2.0
merge-conflicts, v1.4.4
minimap, v4.24.4
minimap-highlight-selected, v4.4.0
open-recent, v5.0.0
pigments, v0.29.2
sass-compiler-autosave, v0.5.0
temporary-root, v1.0.0

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
Does this happen even after a restart? or just after the upgrade?

**Comment by emileber** [comment]
After restart

**Comment by steelbrain** [comment]
Investigating

**Comment by emileber** [comment]
I'm happy to help, don't hesitate to ask for more info

**Comment by steelbrain** [comment]
Gah, typos. Fixed in 1.11.10

**Comment by emileber** [comment]
It works well! Thanks again
